sycl/include/syclcompat/device.hpp

```diff
@@ -545,8 +545,14 @@ Use 64 bits as memory_bus_width default value."
     _queues.clear();
     // create new default queue
     // calls create_queue_impl since we already have a locked m_mutex
-    _saved_queue = _default_queue =
-        create_queue_impl(print_on_async_exceptions, in_order);
+
+    if (in_order) {
+      _saved_queue = _default_queue = create_queue_impl(
+          print_on_async_exceptions, sycl::property::queue::in_order());
+    } else {
+      _saved_queue = _default_queue =
+          create_queue_impl(print_on_async_exceptions);
```

Comment:
Nit: Could be a ternary with an empty property list in the 2nd case...

Reply:
yes, good point